doc/src/manual/profile.md

```diff
@@ -321,6 +327,15 @@ you want to analyze, then call [`Profile.clear_malloc_data()`](@ref) to reset al
  Finally, execute the desired commands and quit Julia to trigger the generation of the `.mem`
 files.
 
+### GC Logging
+
+While [`@time`](@ref) logs high-level stats about memory usage and garbage collection over the course
+of evaluating an expression, it can be useful to log each garbage collection event, to get an
+intuitive sense of how often the garbage collector is running, how long it's running each time,
+and how much garbage it collects each time. This can be enabled with
+[`Profile.enable_gc_logging(true)`](@ref), which causes Julia to log to stderr every time a
```

Comment:
seems like this ref isn't working… Do I need to add a heading for `Profile.enable_gc_logging` somewhere else (maybe in the `@docs` blocks in `base.md`), and/or refer to it differently (using the type signature, or just the name, etc?) Thanks!

Reply:
Is it fixed now that you fixed it to the right module name?

Reply:
🤔 no, it still seems to have failed... Maybe you don't need the leading `Base.`? Other things don't seem to use it, and somehow they resolve.. Maybe just `Experimental. enable_gc_logging(true)` would be enough? 🤷